### PR TITLE
Mods from notebook run throughs

### PR DESCRIPTION
**Pull request by briochh**
Mostly minor, so far. 

I have moved to use pip -e  in the conda environment file to install the local (distributed in this repo) version of pyemu and flopy. This should avoid the need for the path wrangling at each tutorial run time. The flopy and pyemu version in the repo are current version downloaded from GitHub. We could move link pip straight to a git version of these repos so we could avoid maintaining here. 

+ Replaced some img ref `\`with `/` for compat outside windows. Think this should work in Windows too?

More to come

### CODE REVIEW

**Comment by jtwhite79** [comment]
@briochh I pulled your develop branch locally, recreated the env but something is up with the assert that is checking where flopy and pyemu are located: 
image

I guess this has to do with the pip -e stuff?  Any ideas on this?

**Comment by briochh** [comment]
Yeah. Need to strip out sys path append dependencies.

**Comment by jtwhite79** [comment]
So does this mean those asserts dont work with pip -e? looks like the `__file__` attr is `None`...I like those asserts just because it keeps someone from creating the env, the updating flopy/pyemu, and then telling us that something isnt working!  I may or may not have experienced exactly this issue!

**Comment by briochh** [comment]
Asserts should still work. In fact, I think they kinda are. The problem is that with “../../dependencies” added to the path, python will import the parent flopy and pyemu directories (which contain no source). If you strip out the sys.path manipulation then python should import from the environment location (which is in dependencies/pyemu/pyemu now) and __file__ should still contain “dependencies”

**Comment by jtwhite79** [comment]
doh!  you are right.  I see it now - working as expected if I comment out the sys path insert

**Comment by briochh** [comment]
We can always change it back. Was an attempt to simplify, maybe it doesn’t.

**Comment by jtwhite79** [comment]
I'm good with the pip -e approach - I think the implied best practices are an important piece of these notebooks and it seems like pip -e with conda/mamba (and esp conda pack) is cleaner...

File: environment.yml
Comment (rhugman) [inline environment.yml:1]: should we update python version while we are at it?

Comment (jtwhite79) [reply environment.yml:1]: like 3.10 or 3.11?

Comment (briochh) [reply environment.yml:1]: should work. pandas changes are more affecting than python changes!

**Comment by briochh** [comment]
@jtwhite79, @rhugman, this should be g2g now. Nothing too major. 

- pstfrom sneakpeak notebook added in Part1, just introducing main components
- @jtwhite79 added the install checker notebook
- flopy and pyemu deps have been updated, may update pyemu again once we get a release out (hopefully tomorrow)
- PESTPP updated to 5.2.6 -- Have run through most but need to double check slower (loc, opt, mou) notebooks.
- as mentioned above, dependencies now install as a `pip` step in the `condo`env (although still pointing pip to those local version).
- python version in env updated to 3.10 
- Added some cross ref links to notebooks and caught some spellings.

**Pull request merged**